src/cljs/org/broadinstitute/firecloud_ui/page/workspace/summary/tab.cljs

```diff
@@ -63,7 +63,7 @@
         [comps/SidebarButton {:style :light :margin :top :color :exception-red
                               :text "Delete" :icon :trash-can
                               :disabled? (if locked? "This workspace is locked")
-                              :onClick #(when (js/confirm "Are you sure?")
+                              :onClick #(when (js/confirm "Are you sure?\nBucket data will also be deleted")
```

Comment:
Nit: It'd be good to standardize on punctuation. Did you try this with a period to see how it looked?